agent-core/src/main/java/com/newrelic/agent/android/logging/LogReporting.java

```diff
@@ -163,13 +162,17 @@ public void log(LogLevel logLevel, String message) {
         }
 
         public void logThrowable(LogLevel logLevel, String message, Throwable throwable) {
-            StringWriter sw = new StringWriter();
 
             message = validator.validate(message);
-            throwable = validator.validate(throwable);
-            throwable.printStackTrace(new PrintWriter(sw));
+            if (null == throwable) {
+                logToAgent(logLevel, message);
+                return;
```

Comment:
I am just passing the message if the throwable is null. We don't need to validate the message because it is already getting validated in the logToAgent method.